CONTRIBUTING.md

```diff
@@ -14,3 +14,18 @@ Please review the guidelines below before opening a PR.
     2. Run formatting tests locally with `black {source_file_or_dir}`, or, after installing all jiant requirements and activating the environment, you can run `pre-commit install` so that black checking and reformatting runs automatically before committing your staged changes.
     3. For involved changes, run a minimal experiment (e.g., with “max_vals=1 and val_interval=10”) to check that your changes don’t break at runtime.
 6. Once your PR is ready for review, in your Draft PR press “Ready for review”. This will invite code owners to provide reviews, and makes the branch mergeable.
+
+### Test and documentation guidelines
```

Comment:
Be consistent about colon vs. no-colon after ###

Reply:
👍 (done in 66e28d1)